anaDACScan.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+
+r"""
+``anaDACScan`` -- analyzes DAC scan data
+========================================
+
+Synopsis
+--------
+
+**anaDACScan.py** [*OPTIONS*]
+
+Description
+-----------
+
+This script reads in calibration information and DAC scan data, performs a fit for each VFAT, computes the DAC value corresponding to the nominal current or voltage for each VFAT, and reports the results.
+
+
+Arguments
+---------
+
+.. program:: anaDACScan.py
+
+.. option:: infilename
+
+    Name of the input file root.
+
+.. option:: --assignXErrors  
+
+    Use the dacValX_Err values stored in the input file
+
+.. option:: --calFileList
+  
+    Provide a file containing a list of calFiles in the following format. Example:: 
+
+    OH1 calFileOH1
+    OH2 calFileOH2
+    OH3 calFileOH3
+    etc
+
+.. option:: --outfilename
+
+    Name of the output root file. Default is DACFitData.root.
+
+Example
+-------
+
+.. code-block:: bash
+
+    anaDACScan.py /afs/cern.ch/user/d/dorney/public/v3Hack/dacScanV3.root  --calFileList calibration.txt --assignXErrors
+
+
+Environment
+-----------
+
+
+Internals
+---------
+
+
+"""
+
+if __name__ == '__main__':
+
+    import ROOT as r
+    
+    from gempython.gemplotting.utils.anautilities import parseCalFile
+    from gempython.utils.nesteddict import nesteddict
+    from gempython.gemplotting.utils.anautilities import make3x8Canvas
+    
+    from gempython.gemplotting.mapping.chamberInfo import chamber_config
+
+    import os
+    
+    import argparse
+    
+    parser = argparse.ArgumentParser(description='Arguments to supply to anaDACScan.py')
+
+    parser.add_argument('infilename', type=str, help="Filename from which input information is read", metavar='infilename')
+    parser.add_argument("--calFileList", type=str, help="File specifying which calFile to use for each OH. Format: OH1 filenameOH1 <newline> OH2 filenameOH2 <newline> etc", metavar="calFileList")
```

Comment:
Again it's not clear that `OH1` should be an integer from this `helpstring`.

Reply:
fixed